src/LayoutManager/LayoutManager.ts

```diff
@@ -30,6 +30,40 @@ export type SerializedLayoutManager = {
   strategy: string;
 };
 
+export const buildStandardEvents = (object: FabricObject, target: Group) => {
```

Comment:
shouldn't this method be part of the class??
why did you decide not to have there?